src/test/scala/pureconfig/PureconfSuite.scala

```diff
@@ -611,6 +612,7 @@ class PureconfSuite extends FlatSpec with Matchers with OptionValues with TryVal
     loadConfig[ConfWithConfigList](emptyConf).failure.exception shouldEqual KeyNotFoundException("conf")
     loadConfig[ConfWithDuration](emptyConf).failure.exception shouldEqual KeyNotFoundException("i")
     loadConfig[SparkNetwork](emptyConf).failure.exception shouldEqual KeyNotFoundException("timeout")
+    loadConfig[ConfWithDuration](emptyConf).failure.exception shouldEqual KeyNotFoundException("i")
```

Comment:
This line seems repeated from [L613](https://github.com/melrief/pureconfig/pull/60/files#diff-e8e47c5f8227db17829b61b757b0ee0eR613).